### ISSUE TITLE
Visualization of the communities in GraphRicciCurvature Tutorial

### ISSUE THREAD

**Issue by ElahehKarampour** [saibalmars/GraphRicciCurvature#40]
Hello,
I'm new to GraphRicciCurvature library and want to reproduce the code in GraphRicciCurvature Tutorial with my own graph.
I don't know what is the clustering label in the bellow code:

```
def draw_graph(G, clustering_label="club"):
    """
    A helper function to draw a nx graph with community.
    """
    complex_list = nx.get_node_attributes(G, clustering_label)

    le = preprocessing.LabelEncoder()
    node_color = le.fit_transform(list(complex_list.values()))

    nx.draw_spring(G, nodelist=G.nodes(),
                   node_color=node_color,
                   cmap=plt.cm.rainbow,
                   alpha=0.8)

```

Is it possible to use the ricci comunity detection for unsupervised problem? 
We don't have any information about the number of communities in my network.

Thank you.

**Comment by saibalmars** [comment]
Hi, here the cluster label is just the ground truth provided from dataset, and mostly used to evaluate the clustering performance, if you don't have ground truth you can still run the code with no problem by assigning all nodes with a fake attribute (like all 0 for all node).
